k-distribution/tutorial/1_k/1_lambda/lesson_5/tests/arithmetic-div-zero.lambda.out

```diff
@@ -1 +0,0 @@
-V ==K 1 /Int 0
```

Comment:
Why was this test deleted? Because llvm backend crashes on this output?

Reply:
Yes